addons/languages/urdu/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
this should be set to `false`, until we create the app in the play-store

Reply:
Aren't we going to create the app in the Play Store?

Reply:
yes, we will, but only after this gets merged into master.

Reply:
Changed to `false`.